README.md

```diff
@@ -100,11 +100,16 @@ class MyComponent extends Component {
 }
 ```
 
-Options:
+In order to prevent the "Warning: Unknown props" message, you can use the `propsNamespace` option:
 
 ```js
-{
-  propsNamespace: 'translator', // injected props will be set in props.translator
+@withTranslator({ propsNamespace: 'translator' })
```

Comment:
Ideally we could even export a `translatorShape` that users can import, like `react-router` does: https://github.com/ReactTraining/react-router/pull/3349/files.

@iamdey : should I open a "wishlist" issue about this?